website/docs/components/form/file-input/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: File Input
+description: A File Input that enables users to select one or more files from their local device for upload.
```

Comment:
This doesn't follow the pattern of the other component descriptions and it shouldn't be capitalized here actually since it's explaining what the component is.

Reply:
How about this instead:

"A form input that enables users to select one or more files from their local device for upload."